package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@acpr/rate-limit-postgresql",
-  "version": "1.3.2",
```

Comment:
I usually do this separately by running `npm version minor` (or `major` or `patch`) on the main branch after the PR is merged in. 

The npm command creates both a commit and a git tag, and then you can push both with `git push --follow-tags`. The tag makes it much easier to find the code for a specific version later on.